backend/tests/core/DatabaseHelper.cs

```diff
@@ -19,11 +19,38 @@ namespace Pims.Core.Test
     public static class DatabaseHelper
     {
         #region Methods
+        /// <summary>
+        /// Creates an instance of a PimsContext and initializes it with the specified 'permission'.
+        /// Uses an InMemoryDatabase instead of relational.
+        /// </summary>
+        /// <param name="helper"></param>
+        /// <param name="permission"></param>
+        /// <param name="ensureDeleted"></param>
+        /// <returns></returns>
+        public static PimsContext CreatePimsContext(this TestHelper helper, Permissions permission, bool ensureDeleted = false)
+        {
+            return helper.CreatePimsContext(StringHelper.Generate(10), permission, ensureDeleted);
+        }
+
+
         /// <summary>
         /// Creates an instance of a PimsContext and initializes it with the specified 'user'.
         /// Uses an InMemoryDatabase instead of relational.
         /// </summary>
         /// <param name="helper"></param>
+        /// <param name="user"></param>
+        /// <param name="ensureDeleted"></param>
+        /// <returns></returns>
+        public static PimsContext CreatePimsContext(this TestHelper helper, ClaimsPrincipal user, bool ensureDeleted = false)
```

Comment:
By default it will generate an unique PimsContext and add it to the service provider.  This reduces the amount of boilerplate code in tests.